README.rst

```diff
@@ -38,7 +38,7 @@ Community
 
 You can talk to the community through:
 
-* `@pybeeware on Twitter`_
+* `@beeware@fosstodon.org on Mastodon`_
 
 * BeeWare Gitter channel: `beeware/general`_
```

Comment:
Since you're in the area - updating the references to Gitter to Discord would also be worthwhile.